src/App.js

```diff
@@ -1,13 +1,24 @@
 import { useState } from "react";
 import "./App.css";
+import "./constants/utility.css";
 import { Footer, Header, Main, Sidebar } from "./pageLayout";
+import { useLocation } from "react-router-dom";
 
 function App() {
   const [showSidebar, setShowSidebar] = useState(false);
+  const forbiddenPages = ["/login", "/signup", "*"];
+  const currentLocation = useLocation().pathname;
+  console.log(currentLocation);
```

Comment:
you can remove console logs before pushing changes

Reply:
Thanks, I missed it